### PR TITLE
Copy-Editing/Grammatical Fixes: Chapters 2-13

### PR DESCRIPTION
**Pull request by ericoulster**
I made small sentence structure changes throughout the course to improve clarity.

Issues are relatively small, and usually involve less than a sentence of revisions across a handful of modules.

### CODE REVIEW

**Review by mmhamdy — APPROVED** [review]
Looks great! Thanks for the fixes 🙂

**Comment by sergiopaniego** [comment]
I can't approve the changes but they look good to me 😄

**Review by sergiopaniego — APPROVED** [review]

**Review by johko — APPROVED** [review]
Thanks for the fixes :+1: LGTM

File: chapters/en/unit3/vision-transformers/vision-transformer-for-object-detection.mdx
Comment (johko) [inline chapters/en/unit3/vision-transformers/vision-transformer-for-object-detection.mdx:1]: I just saw that one of the workflows failed and saw that you corrected the name of this file. You will also need to change it in the _toctree.yml, located at chapters/en/_toctree.yml

**Review by johko — CHANGES_REQUESTED** [review]
Spotted a little problem in hindsight, I left a comment how to fix it

File: chapters/en/unit3/vision-transformers/vision-transformer-for-object-detection.mdx
Comment (ericoulster) [reply chapters/en/unit3/vision-transformers/vision-transformer-for-object-detection.mdx:1]: Makes sense. 

I pushed the change you mentioned and the automated tests appear to be green now.

**Review by johko — APPROVED** [review]
Thanks for the quick response :+1:

**Pull request merged**